java/build.gradle

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * user guide available at https://docs.gradle.org/5.1.1/userguide/tutorial_java_projects.html
+ */
+
```

Comment:
Minor - remove extra blank lines in this and other files.

Reply:
Can users get these gradle files from gradle download/install?  If we include them as part of our sample, how are going to maintain them moving forward?

Reply:
yes. Gradle takes care of automatically downloading the dependency.